deploy_slave.sh

```diff
@@ -78,6 +78,9 @@ getent group piwheels || groupadd piwheels
 getent passwd piwheels || useradd -g piwheels -m -s /bin/bash piwheels
 passwd -d piwheels
 
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | CARGO_HOME=/usr/local sh -s -- -y
+echo "CARGO_HOME=/usr/local/bin" > /etc/environment.d/20-rustup.conf
```

Comment:
Different approach, not changing `CARGO_HOME` but adding Rust binaries to PATH, as suggested by rustup:
```suggestion
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
echo '. ~/.cargo/env' >> /home/piwheels/.bashrc
```

Reply:
Looks like I misses that this script runs aa root, so rustup needs to be installed as piwheels user. A `sudo -u piwheels sh` should then work.